OHHTTPStubs/Sources/OHHTTPStubs.m

```diff
@@ -44,6 +44,7 @@ @interface OHHTTPStubsProtocol : NSURLProtocol @end
 @interface OHHTTPStubs()
 + (instancetype)sharedInstance;
 @property(atomic, copy) NSMutableArray* stubDescriptors;
+@property(atomic, copy) NSNumber* enabledStateNumber;
```

Comment:
Why not using a `BOOL`, as it will be initialized to `YES` at init time so you don't really need the `YES/NO/nil` tri-state anyway?